testkit/src/main/scala/ch/epfl/scala/bsp/testkit/client/mock/MockSession.scala

```diff
@@ -28,7 +29,7 @@ case class MockSession(
       .create()
 
     val listening = launcher.startListening
-    new Thread (() => listening.get()).start()
+    new Thread (() => Try(listening.get())).start()
```

Comment:
Aren't we ignoring any exception thrown here? Why is this necessary?

Reply:
Any exception thrown here will automatically impact the result of a test, therefore I don't think handling them is necessary. It is, however, wrapped in a `Try` to prevent the output of the `CancellationException` whenever the connection is shutdown, since this exception is irrevelant for the user

Reply:
Then the ignoring should be explicit. Let’s replace it with a try catch expression, it’s clearer

Reply:
Makes sense, changed!